FN1 fluency addition

AbbyANoble opened the pull request:
Addresses some of #513. I added a few domain fluency problems at the end. Once we like the choices, we can add something similar to CheckIt.

tdegeorge commented on source/precalculus/source/02-FN/01.ptx:
```diff
@@ -956,6 +956,49 @@
+<activity xml:id="FN1-domain">
+    <introduction>
+      <p>
+       Find the domain of each of the following functions. Write your answer in interval notation
```
```suggestion
       Find the domain of each of the following functions. Write your answer in interval notation.
```

tdegeorge commented:
I saw in the discussion about adding in range. Did we want to add in finding the range of a function from an equation? I don't normally have students find range at this point, but it could be something we have in there and people can choose to use it (or not).

tdegeorge commented:
Should we change [[Activity 2.1.21](https://c9a98b28.tbil.pages.dev/preview/precalculus/instructor/FN1#FN1-determine-domain-from-equation-square-root) 
https://c9a98b28.tbil.pages.dev/preview/precalculus/instructor/FN1#FN1-determine-domain-from-equation-square-root](url) so that it is not just the square root of x? Maybe change that to the square root of 4x-1 or something different? It might help students make connections to the fluency problem part b in the last activity. Also, there are no examples of the square root in the denominator until the fluency question at the end.

AbbyANoble commented:
> I saw in the discussion about adding in range. Did we want to add in finding the range of a function from an equation? I don't normally have students find range at this point, but it could be something we have in there and people can choose to use it (or not).

I don't generally do range from an equation either. Maybe we can make that a non-urgent issue if someone wants to tackle it?

AbbyANoble commented:
> Should we change ([Activity 2.1.21]
> https://c9a98b28.tbil.pages.dev/preview/precalculus/instructor/FN1#FN1-determine-domain-from-equation-square-root) so that it is not just the square root of x? Maybe change that to the square root of 4x-1 or something different? It might help students make connections to the fluency problem part b in the last activity. Also, there are no examples of the square root in the denominator until the fluency question at the end.

Yeah. I'd suggest something like $sqrt{x-2}$ though at first so it's easier to see. I think the first one having a fractional endpoint may be too much to start with! At least for my students.

AbbyANoble commented:
I changed up the $\sqrt{x}$ problem if you want to check it out.

tdegeorge commented:
> > Should we change ([Activity 2.1.21]
> > https://c9a98b28.tbil.pages.dev/preview/precalculus/instructor/FN1#FN1-determine-domain-from-equation-square-root) so that it is not just the square root of x? Maybe change that to the square root of 4x-1 or something different? It might help students make connections to the fluency problem part b in the last activity. Also, there are no examples of the square root in the denominator until the fluency question at the end.
> 
> Yeah. I'd suggest something like x − 2 though at first so it's easier to see. I think the first one having a fractional endpoint may be too much to start with! At least for my students.
> 
> > Also, there are no examples of the square root in the denominator until the fluency question at the end.
> 
> Do you think that's something they could put together after seeing that denominators and square roots affect the domain? What if we added in a hint to it rather than a full exercise? I feel like it's getting so long already!

I like the hint idea!

tdegeorge commented:
I think we should probably change the wording so that it matches in the following activities as well ("Which of the following values of would cause y to be undefined (if any)?" rather than saying "error").

And then adding the hint in the last problem would be good I think!

AbbyANoble commented:
> I think we should probably change the wording so that it matches in the following activities as well ("Which of the following values of would cause y to be undefined (if any)?" rather than saying "error").
> 
> And then adding the hint in the last problem would be good I think!

I changed the wording and added the hint in! Ready to approve and merge if you are good with this!
